INSTALL.md

```diff
@@ -70,6 +70,15 @@ automatically try and compile the `protoc` in third_party if you cloned the
 repository recursively and it detects that you don't already have it
 installed.
 
+If it hasn't been installed, you can run the following commands to install it.
+
+```sh
+$ cd grpc/third_party/protobuf
+$ sudo make install   # 'make' should have been run by core grpc
+```
+
+Alternatively, you can download `protoc` binaries from
```

Comment:
I suggest leaving this out. Version conflict between installed protoc and protobuf submodule has been a big source of confusion and unnecessary issues get filed.

Reply:
Should I remove both "make install" as well as download options?

Reply:
make install will install the protoc that we want (the issue is the potential version mismatch between third_party/protobuf submodule and protoc downloaded by the user). Please leave it as is. Just the download options may cause confusion.